mailpoet/lib/Segments/WooCommerce.php

```
@@ -276,7 +301,7 @@ private function markRegisteredCustomers(): void {
     ", ['capabilities' => $wpdb->prefix . 'capabilities', 'source' => Source::WOOCOMMERCE_USER]);
   }
 
-  private function insertSubscriberFromOrder(\WC_Order $wcOrder, string $status): ?string {
+  private function insertSubscriberFromOrder(\WC_Order $wcOrder, string $status, bool &$wasNewlyCreated = false): ?string {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$wasNewlyCreated'. (undefined)

(UnusedFormalParameter)